### ISSUE TITLE
Remove HomotopyContinuation extension once supported in MTK

### ISSUE THREAD

**Issue by isaacsas** [SciML/Catalyst.jl#1079]
Once https://github.com/SciML/ModelingToolkit.jl/pull/3114/files is merged, and updated to include rational functions, we should probably remove the Catalyst HomotopyContinuation extension and instead have users go via MTK.

**Comment by ChrisRackauckas** [comment]
With https://github.com/SciML/ModelingToolkit.jl/pull/3151, my understanding is that it's now feature complete to the Catalyst one? @TorkelE try using it and update the docs, and if it does everything you need, we can remove the Catalyst-specific one.

**Comment by isaacsas** [comment]
We’ll still probably need at least a thin wrapper here to handle conserved equations and directly passing ReactionSystems.  But it would be great if that is all it needs to be going forward.

**Comment by isaacsas** [comment]
Our docs will also need updates if the interface is at all changed after we update to wrap MTK.

**Comment by TorkelE** [comment]
Yes, we still need some additional modifications, but we should be able to make the Catalyst interface more straightforward at least.
